corehq/apps/hqwebapp/templates/hqwebapp/bootstrap5/base_navigation.html

```diff
@@ -61,6 +61,30 @@
       {% if not request.user.is_authenticated %}
         <nav class="navbar-signin ms-2" role="navigation">
           <div class="nav-settings-bar">
+            <div class="navbar-nav dropdown">
+              <button
+                class="dropdown-toggle btn btn-default navbar-btn"
+                data-bs-toggle="dropdown"
+                aria-haspopup="true"
+                aria-expanded="false"
+              >
+                {{ LANGUAGE_CODE }} <span class="caret"></span>
+              </button>
+              <ul class="dropdown-menu">
+                {% for lang in LANGUAGES %}
+                  <li>
```

Comment:
i think bootstrap 5 handles dropdowns slightly differently... requiring a dropdown item class on the child elements
https://getbootstrap.com/docs/5.3/components/dropdowns/#single-button

Reply:
https://github.com/dimagi/commcare-hq/pull/36733/commits/7b63d5b8f9783d42865ff6a83c0b03e606229cf2